Resources/Lang/en_US.json

```diff
@@ -743,7 +743,7 @@
   "ConsigliereInfoLong": "(Impostors):\nAs the Consigliere, you can reveal the roles of other players using your kill button.\n\nSingle click: Reveal role\nDouble click: Kill\n\nIf you run out of reveal uses, your kill button functions normally.",
   "LudopathInfoLong": "(Impostors):\nAs the Ludopath, your kill cooldown is randomized.\n\nMinimum it can be is 1 second, while the maximum is your default kill cooldown.",
   "GodfatherInfoLong": "(Impostors):\nAs the Godfather, you vote someone to make them your target.\nIn the next round, if someone kills the target, the killer will turn into a Refugee.",
-  "ChronomancerInfoLong": "(Impostors):\nAs the Chronomancer, you have a charge bar which indicates when the massacre is ready. When it is at 100% the next time you kill someone, you go into massacre mode, meaning you can kill indefinitely until your bar runs out of charge. Otherwise, you have a normal KCD.",
+  "ChronomancerInfoLong": "(Impostors):\nAs the Chronomancer, you have a charge bar which indicates when the slaughter is ready. When it is at 100% the next time you kill someone, you go into slaughter mode, meaning you can kill indefinitely until your bar runs out of charge. Otherwise, you have a normal KCD.",
```

Comment:
also change it in here 
![image](https://github.com/0xDrMoe/TownofHost-Enhanced/assets/143632280/f86439cb-7e9e-4685-88b2-ffeba1562ac0)

Reply:
Fixed